experiments/download_ssl4eo.py

```diff
@@ -534,10 +622,12 @@ def fix_random_seeds(seed: int = 42) -> None:
     parser.add_argument("--cloud_pct", type=int, default=20)
     parser.add_argument("--num_workers", type=int, default=8)
     parser.add_argument("--log_freq", type=int, default=10)  # print frequency
-    parser.add_argument("--preview", action="store_true")
     parser.add_argument(
         "--resume", type=str, default=None
     )  # resume from existing coordinates
+    parser.add_argument(
+        "--overlap_check", type=str, default="rtree", choices=["grid", "rtree", None]
```

Comment:
Is there any reason we would want to support both? Seems like the rtree implementation is more technically correct and superior in every way. This file is already gigantic, so we should try to minimize how many lines of code we have to maintain. I would vote for deleting the old logic.

Reply:
Wasn't very sure, as it is the original implementation and in terms of speed grid is a bit faster. But yes it's not very necessary here, let me delete it.